app/livechat/server/lib/screenSharing/providerScripts/cobrowse.txt

```diff
@@ -0,0 +1,59 @@
+const loadSDKScript = () => {
```

Comment:
Why `cobrowse.txt` instead of `cobrowse.js` ?
Also, why did you create another folder `providerScripts` instead of creating a folder inside the `provider` folder?
Something like `/providers/scripts`.